### ISSUE TITLE
Accessibility issue: Page should contain a level-one heading on /primer/view_components/security/code-scanning

### ISSUE THREAD

**Issue by lindseywild** [lindseywild/accessibility-scanner#112]
## What
An accessibility scan flagged the element `<html lang="en" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark" data-a11y-animated-images="system" data-a11y-link-underlines="true" class="js-focus-visible" data-js-focus-visible="" data-turbo-loaded="">` on https://github.com/primer/view_components/security/code-scanning?query=is%3Aclosed+branch%3Amain because page should contain a level-one heading. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/page-has-heading-one?application=playwright.

## Screenshot

A screenshot was captured when this issue was detected.

![Screenshot](https://raw.githubusercontent.com/lindseywild/accessibility-scanner/gh-cache/.screenshots/a9be211a-d576-4ed7-9a9b-c9fc9bdd7af9.png)

> Screenshot ID: `a9be211a-d576-4ed7-9a9b-c9fc9bdd7af9`
> [View in gh-cache branch](https://github.com/lindseywild/accessibility-scanner/blob/gh-cache/.screenshots/a9be211a-d576-4ed7-9a9b-c9fc9bdd7af9.png)

To fix this, ensure that the page, or at least one of its frames contains a level-one heading.

Specifically:

Fix all of the following:
-   Page must have a level-one heading

## Acceptance Criteria
- [ ] The specific axe violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
